pygmt/tests/test_datasets_moon_relief.py

```diff
@@ -0,0 +1,58 @@
+"""
+Test basic functionality for loading Moon relief datasets.
+"""
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pygmt import config
+from pygmt.datasets import load_moon_relief
+from pygmt.exceptions import GMTInvalidInput
+
+
+def test_moon_relief_01d():
+    """
+    Test some properties of the Moon relief 01d data.
+    """
+    with config(GMT_DATA_SERVER="candidate"):
+        data = load_moon_relief(resolution="01d")
```

Comment:
No need to set GMT_DATA_SERVER if the data is released：
```suggestion
    data = load_moon_relief(resolution="01d")
```